src/Workspace.scss

```diff
@@ -824,3 +824,24 @@ $color-map: (
 .btn:disabled.opacity-none {
   opacity: 1;
 }
+
+.max-100 {
```

Comment:
we have .height-100, please rename to max-h-100 or max-height-100 to avoid confusion with max-width